solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -318,6 +323,37 @@ contract InterchainAccountRouter is Router {
         }
     }
 
+    function moduleType() external pure returns (uint8) {
+        return uint8(IInterchainSecurityModule.Types.ROUTING);
+    }
```

Comment:
inherit `AbstractRoutingIsm` and implement `route` which the relayer introspects

Reply:
Done